app/controllers/menus_controller.rb

```diff
@@ -0,0 +1,32 @@
+class MenusController < ApplicationController
+  def index
+    @menus = Menu.all
+  end
+
+  def new
+    authorize Menus::NewFacade.new.menu
+    @facade = Menus::NewFacade.new
+    redirect_to root_path, flash: {warning: t('menus_messages.new.messages.weekends')} if
```

Comment:
move `redirect_to root_path, flash ...` to private methods and call them here. Two-line ifs look messy